src/main/java/core/basesyntax/CalculateArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface CalculateArea {
```

Comment:
Ivan's comment not fixed. Interface names should be nouns or adjectives
```suggestion
public interface AreaCalculator {
```